C23_WPDE_Mobile_01

Sperling-0 opened the pull request:
- Prepare campaign for C23_WPDE_Mobile_01
- Implement VAR for C23_WPDE_Mobile_01

Ticket: https://phabricator.wikimedia.org/T348625

Sperling-0 committed:
Prepare campaign for C23_WPDE_Mobile_01

Ticket: https://phabricator.wikimedia.org/T348625

Sperling-0 committed:
Implement VAR for C23_WPDE_Mobile_01

Ticket: https://phabricator.wikimedia.org/T348625

Abban reviewed (changes requested):
The back button is missing from step 2 of both banners, if you look in the BannerCtrl on the  wikipedia.org mobile banner you can see that the `UpgradeToYearlyButtonForm` Takes in a template for it to put into a slot:

```html
<template #[FormStepNames.UpgradeToYearlyFormStep]="{ pageIndex, submit, isCurrent, previous }: any">
	<UpgradeToYearlyButtonForm :page-index="pageIndex" @submit="submit" :is-current="isCurrent" @previous="previous">
		<template #back>
			<ChevronLeftIcon/> {{ $translate( 'back-button' ) }}
		</template>
	</UpgradeToYearlyButtonForm>
</template>
```

Other than that it all looks good to me

Abban committed:
Fix snags on C23_WPDE_Mobile_01

Ticket: https://phabricator.wikimedia.org/T348625

Abban committed:
Add missing event tracking to C23_WPDE_Mobile_01

This banner was missing some events.

Ticket: https://phabricator.wikimedia.org/T350151

Abban committed:
Hide Soft Close on final impression on WPDE Mobile

Ticket: https://phabricator.wikimedia.org/T349186
